grimoiressg/arguments.py

```diff
@@ -0,0 +1,31 @@
+import argparse
+
+
+def parse_arguments_to_initial_context():
+    parser = argparse.ArgumentParser(
+        description='''
+            Grimoire is a minimalistic Static Site Generator.
+            In the simplest case the only argument needed is at least one content file. \
+            The rest of the flags is used to customize the behavior.
+        '''
+    )
+    parser.add_argument("content_file", nargs='+', help="one or more content files")
```

Comment:
Why are you using single quotes in the one place but double quotes in other places?

Reply:
Copy-paste error from the documentation. ^^"